openedx2zim/scraper.py

```diff
@@ -11,15 +13,16 @@
 import tempfile
 import urllib
 import uuid
-import concurrent.futures
+from gettext import gettext as _
```

Comment:
you should import it from `zimscraperlib.i18n`

Reply:
Got it.